src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/corporates/CorporateResource.kt

```diff
@@ -86,15 +148,25 @@ class CorporateResource(private val corporateService: CorporateService) {
   ): CorporateOrganisation = corporateService.getCorporateById(corporateId)
 
   @PreAuthorize("hasRole('ROLE_NOMIS_CONTACTPERSONS')")
-  @GetMapping("/corporates/ids")
```

Comment:
Sorry the move has made it look like this new method is change of the GET method